### PR TITLE
Corregido link a Gobernanza

### PR DESCRIPTION
**Pull request by NPalopoli**
Corregido link a On the Governance of The Carpentries

### CODE REVIEW

File: _posts/2022/09/2022-09-16-ec-elections_es.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ El Consejo Ejecutivo de The Carpentries es el órgano de gobierno de la organiza
 
 Puedes encontrar más información sobre el propósito y las actividades del Consejo Ejecutivo en la [sección del Consejo Ejecutivo de nuestro manual](https://docs.carpentries.org/topic_folders/governance/executive-council.html).
 
-Además, puedes leer [The Structure of The Carpentries Executive Council](https://carpentries.org/blog/2018/09/executive-committee-structure/), and [On the Governance of The Carpentries](​​https://carpentries.org/blog/2021/07/carpentries-governance/).
+Además, puedes leer [The Structure of The Carpentries Executive Council](https://carpentries.org/blog/2018/09/executive-committee-structure/), and [On the Governance of The Carpentries](https://carpentries.org/blog/2021/07/carpentries-governance/).
```
Comment (tobyhodges) [inline _posts/2022/09/2022-09-16-ec-elections_es.md]:
```suggestion
Además, puedes leer [The Structure of The Carpentries Executive Council](https://carpentries.org/blog/2018/09/executive-committee-structure/), y [On the Governance of The Carpentries](https://carpentries.org/blog/2021/07/carpentries-governance/).
```

**Review by tobyhodges — CHANGES_REQUESTED** [review]
Thank you for noticing this slightly strange error @NPalopoli! The same link is broken in the versions of this post in other languages too, so I will fix it there as well. I made one suggestion (to be approached with caution, due to lack of Spanish knowledge), because it seems like the word "and" was missed in the translation too? If you have time, please can you check and approve the suggested change/provide a correct translation?

**Review by NPalopoli — COMMENTED** [review]
Nice catch!

**Comment by tobyhodges** [comment]
Thanks for your help. I have now fixed in the English version too.

**Review by tobyhodges — APPROVED** [review]

**Pull request merged**